tests/test_validate_call.py

```diff
@@ -1120,10 +1235,12 @@ def foo():
         class M2(BaseModel):
             z: int
 
-        M = M2
+        M = M2  # noqa: F841
 
-        @validate_call
-        def bar(m: M) -> M:
+        @validate_call(validate_return=True)
+        def bar(m: 'M') -> 'M':
```

Comment:
Why is this change necessary?

Reply:
To my understanding, we only use the namespaces to evaluate types when `str` is used to annotate things or `from __future__ import annotations` is used. Otherwise the annotations are normal Python objects and we do not evaluate them at all. Considering the purpose of this test, I would say the quotes are necessary to tell if namespaces are working correctly.